editor/src/components/canvas/canvas-strategies/strategies/ancestor-metastrategy.spec.browser2.tsx

```diff
@@ -528,7 +528,8 @@ describe('finds an applicable strategy for the nearest ancestor', () => {
     }))
 })
 
-describe('Fragments are transparent for ancestor metastrategy', () => {
+// FIXME Fragments are triggering regular strategies but probably never should be
+xdescribe('Fragments are transparent for ancestor metastrategy', () => {
```

Comment:
I'm nervous about merging and forgetting this xdescribe. I'm pretty sure this is a meaningful test, it may be worth it to block this PR until we figure out how to resolve the issue

Reply:
I disagree about blocking the PR because the issue is already there, it just doesn't happen to show in the specific sample project used in the test.

Reply:
I've adjusted the escape hatch strategy to target the children of child-affective elements, therefore skipping over this problem, so the test has been re-enabled. As mentioned though, this problem still exists, and should be tackled separately to this PR.